health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/securityPopUp.js

```
@@ -21,13 +27,41 @@ const SecurityPopUp = ({ onClose }) => {
+  useEffect(() => {
+    if (!mutation.isLoading && mutation.data) {
+      onClose(); // Close popup after saving
+    }
+  }, [mutation.data, mutation.isLoading, onClose]);
+
   return (
     <PopUp
       onClose={onClose}
       heading={t(`HCM_MICROPLAN_VILLAGE_SECURITY_LEVEL_LABEL`)}
       onOverlayClick={onClose}
       children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)